src/utils/github_updater.rs

```diff
@@ -21,15 +21,15 @@ pub fn github_updater() -> Result<()> {
     //       always the same across all versions of a crate
     for row in &conn.query(
         "SELECT DISTINCT ON (crates.name)
-                                        crates.name,
-                                        crates.id,
-                                        releases.repository_url
-                                 FROM crates
-                                 INNER JOIN releases ON releases.crate_id = crates.id
-                                 WHERE releases.repository_url ~ '^https*://github.com' AND
-                                       (crates.github_last_update < NOW() - INTERVAL '1 day' OR
-                                        crates.github_last_update IS NULL)
-                                 ORDER BY crates.name, releases.release_time DESC",
+                crates.name,
+                crates.id,
+                releases.repository_url
+         FROM crates
+         INNER JOIN releases ON releases.crate_id = crates.id
+         WHERE releases.repository_url ~ '^https*://github.com' AND
```

Comment:
This regex doesn't look right ... it matches `https*` but instead it should match `https?`. Currently it will accept `httpsssssssssssssssss` when it shouldn't.